ksqldb-engine/src/test/java/io/confluent/ksql/engine/rewrite/QueryAnonymizerTest.java

```diff
@@ -65,18 +65,6 @@ public void shouldLeaveListAndPrintStatementsAsTheyAre() {
     Assert.assertEquals("LIST QUERIES;", anon.anonymize("LIST QUERIES;"));
   }
 
-  @Test
-  public void describeStatementsShouldGetAnonymized() {
```

Comment:
Hit hits, too. Fixed by removing none-supported `EXTENDED` keyword (like you did above) instead of dropping the test.